src/scss/_variables.scss

```diff
@@ -0,0 +1,10 @@
+$main-font: 'Open Sans', sans-serif;
+$accent-font: 'Sanchez', serif;
+
+$dark-grey: #263238;
+$light-grey: #f5f5f5;
+$gold: #ffca28;
+$dark-blue-purp: #311b92;
+$medium-grey: #eceff1;
+$rich-purp: #4a148c;
+$green: #30BD9E;
```

Comment:
I think it's fine for now since the variable names are integrated in other spots, but if we get to it, I think we should rename these things that aren't specific to the actual colors, e.g. "$main-color", "$accent-light-color", etc.